app/tests/containers/Forms/Form.test.tsx

```diff
@@ -100,13 +106,19 @@ describe('Form', () => {
               ]
             },
             formJsonByFormId: {
-              formJson: emissionForm
+              formJson: emissionForm,
+              ciipApplicationWizardByFormId: {
+                formPosition: 1
+              }
             }
           }
         }}
       />
     );
     expect(r).toMatchSnapshot();
+    expect(r.find('.form-submit a').length).toBe(1);
+    expect(r.find('Button').length).toBe(2);
+    expect(r.find('Button.btn-secondary').text()).toBe('Back');
```

Comment:
Looks good. I'll merge this now, but I'll rewrite it in a separate unit test (using shallow render) in #205 , as I changed this file to an integration test